javase/src/main/java/com/google/zxing/client/j2se/CommandLineEncoder.java

```diff
@@ -47,8 +51,35 @@ public static void main(String[] args) throws Exception {
       outFileString += '.' + config.imageFormat.toLowerCase(Locale.ENGLISH);
     }
     
-    BitMatrix matrix = new MultiFormatWriter().encode(
+    BitMatrix matrix = null;
```

Comment:
Yikes, this is a lot of extra code to special case one arg to one encoder. Why not handle this via the existing hint? It wouldn't hurt to let the code downstream parse an String if needed to make this work. No need for an arbitrary mapping to int